src/createServer.js

```diff
@@ -1,3 +1,64 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+/* eslint-disable max-len */
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+const createServer = () => {
+  const availableCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const url = req.url.split('?');
+
+    const originalText = url[0].slice(1);
+
+    const params = new URLSearchParams(url[1]);
+    const targetCase = params.get('toCase');
+
+    if (!originalText || !targetCase || !availableCases.includes(targetCase)) {
+      const errorResponse = {
+        errors: [],
+      };
+
+      if (!originalText) {
+        errorResponse.errors.push({
+          message: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+        });
+      };
```

Comment:
Instead of duplicating checks like `!availableCases.includes(targetCase)` just check for `errorResponse.errors !== 0` afterwards
```suggestion
      const errorResponse = {
        errors: [],
      };

      if (!originalText) {
        errorResponse.errors.push({
          message: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
        });
      };
```